static/javascripts/components/dashboard.jsx

```diff
@@ -69,7 +69,7 @@ var MainDiv = React.createClass({
         return (
         <div className="mainDiv ">
             {itemNodes}
-            <SurveyCreationCard/>
+            <SurveyCreationCard user_data={this.props.data}/>
```

Comment:
@Bradyta take a look at this. I want to pass the initial data to the creation card to take the user name, but is not working.